src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div>
```

Comment:
Don't use classes like that, here you just didn't apply class to a block to not give it a margin - it's wrong approach. You have `:first-child`, `:last-child` and `:nth-child` pseudoclasses, use them to add/remove styles for n-th elements
```suggestion
        <div class="form">
```